backbone/src/main/java/org/researchstack/backbone/step/QuestionStep.java

```diff
@@ -40,10 +40,10 @@ public QuestionStep(String identifier) {
      *
      * @param identifier The identifier of the step (a step identifier should be unique within the
      *                   task).
-     * @param title      A localized string that represents the primary text of the question.
+     * @param question      A localized string that represents the primary text of the question.
      */
-    public QuestionStep(String identifier, String title) {
-        super(identifier, title);
+    public QuestionStep(String identifier, String question) {
+        super(identifier, question);
```

Comment:
do not change the method signature.. if it doesn´t works anymore for us.. we should deprecate  this method and add a comment explaining the title will be used as a question. And add another constructor with the required info